src/main/java/com/synopsys/integration/alert/configuration/ApplicationConfiguration.java

```diff
@@ -93,6 +95,13 @@ public TaskScheduler taskScheduler() {
         threadPoolTaskScheduler.setPoolSize(Runtime.getRuntime().availableProcessors());
         return threadPoolTaskScheduler;
     }
+
+    //TODO: Required during 6.11.0 performance investigation but should be removed in the future in favor of a thread pool
+    @Bean
+    public TaskExecutor taskExecutor() {
+        return new SyncTaskExecutor();
+    }
+
```

Comment:
There are potential issues when we are running without Sync (too many messages distributed can lead to distribution channel failures). After speaking with Paulo, we will want to add this into master for the time being while we work on other performance improvements.

This should be revisited at a later time.